tests/performance_tests/performance_test_basic.py

```diff
@@ -21,26 +21,144 @@
 relaunchTimeout = 30
 emptyBlockGoal = 5
 
-def waitForEmptyBlocks(node1):
+class blockData():
+    def __init__(self):
+        self.transactions = 0
+        self.cpu = 0
+        self.net = 0
+        self.elapsed = 0
+        self.time = 0
+        self.latency = 0
```

Comment:
Not sure exactly where it would fit best in the data structure, potentially in `blockData`, but it would be good to capture the block number and block id so that we can interact with known blocks and known sequences of blocks.